platforms/hyperledger-besu/charts/node_tessera/templates/deployment.yaml

```diff
@@ -217,47 +227,17 @@ spec:
         command: ["sh", "-c"]
         args:
           - |-
-            apk update && apk add jq curl
             mkdir -p $TESSERA_HOME/logs;
             mkdir -p $TESSERA_HOME/tm;
             DDIR=$TESSERA_HOME/tm;
             PASSWORD=$(cat $TESSERA_HOME/crypto/keys/password);
```

Comment:
Q: Why is this section removed?

Reply:
`tessera` container is not dependent on any of these binaries, the dependencies are separated into init containers